spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.spec.js

```diff
@@ -96,12 +96,16 @@ describe('GA4 click tracker', function () {
     beforeEach(function () {
       attributes = {
         event_name: 'navigation',
-        type: 'a link'
+        type: 'a link',
+        index: 1
```

Comment:
Should this test reflect the single digit edge case for links? I.e. should we change this to `index: {index_link: 1}`?

Reply:
Have updated now, thanks.